db/rebuildSubjectHierarchy.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * db/rebuildSubjectHierarchy.js
+ */
```

Comment:
Can you add some description here about why an admin would ever need to run this? Would it be possible to introspect the db and show the bad data it's going to repair? And have a "read-only" mode where it just shows you the bad data, but doesn't actually rebuild the hierarchy?

Reply:
I need to find out that thing if we can find out with the script  about bad data.